<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>464</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>758</width>
    <height>464</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9000</width>
    <height>9000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smart Wind</string>
  </property>
  <property name="windowIcon">
   <iconset resource="terminal.qrc">
    <normaloff>:/images/dronejpg</normaloff>:/images/dronejpg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>39</width>
    <height>39</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QWidget" name="terminalWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Status</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Status</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="Statuslabel">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Mode</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="ModeLabel">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="stopButton">
              <property name="text">
               <string>stop</string>
              </property>
              <property name="icon">
               <iconset resource="terminal.qrc">
                <normaloff>:/images/icons8-stop-96.png</normaloff>:/images/icons8-stop-96.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="uploadButton">
              <property name="text">
               <string>upload and start</string>
              </property>
              <property name="icon">
               <iconset resource="terminal.qrc">
                <normaloff>:/images/icons8-play-100.png</normaloff>:/images/icons8-play-100.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="EvenModeTab">
            <attribute name="title">
             <string>Even Mode</string>
            </attribute>
            <widget class="QWidget" name="gridLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>50</y>
               <width>411</width>
               <height>131</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QSlider" name="EvenmodehorizontalSlider">
                <property name="maximum">
                 <number>4095</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="EvenmodespinBox">
                <property name="maximum">
                 <number>4098</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QCheckBox" name="checkBox_even_auto">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>210</y>
               <width>151</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>automatically upload</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="GustModeTab">
            <attribute name="title">
             <string>Gust Mode</string>
            </attribute>
            <widget class="QWidget" name="">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>40</y>
               <width>410</width>
               <height>109</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="1">
                 <widget class="QSlider" name="gust_upper_limit_slider">
                  <property name="maximum">
                   <number>4095</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Lower Limit :</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="gust_lower_limit_slider">
                  <property name="maximum">
                   <number>4095</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QSpinBox" name="gust_upper_limit_spinbox">
                  <property name="maximum">
                   <number>4095</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QSpinBox" name="gust_lower_limit_spinbox">
                  <property name="maximum">
                   <number>4095</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Upper limit :</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Period (ms) :</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSlider" name="gust_period_slider">
                  <property name="minimum">
                   <number>500</number>
                  </property>
                  <property name="maximum">
                   <number>20000</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QSpinBox" name="gust_period_spinbox">
                  <property name="minimum">
                   <number>500</number>
                  </property>
                  <property name="maximum">
                   <number>20000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>WARNING : upper limit cannot be assigned smaller than lower limit!!! </string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="SheerModeTab">
            <attribute name="title">
             <string>Sheer Mode</string>
            </attribute>
            <widget class="QListView" name="listView">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>20</y>
               <width>111</width>
               <height>231</height>
              </rect>
             </property>
            </widget>
            <widget class="Line" name="line">
             <property name="geometry">
              <rect>
               <x>303</x>
               <y>20</y>
               <width>20</width>
               <height>261</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
            <widget class="QWidget" name="">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>30</y>
               <width>111</width>
               <height>221</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QSpinBox" name="sheer_spinBox">
                <property name="maximum">
                 <number>4095</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sheer_mode_append_button">
                <property name="text">
                 <string>Append</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>Delete last</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sheer_clear_button">
                <property name="text">
                 <string>Clear All</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="">
             <property name="geometry">
              <rect>
               <x>320</x>
               <y>50</y>
               <width>111</width>
               <height>81</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>set at : </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="sheer_comboBox">
                <item>
                 <property name="text">
                  <string>Row</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Column</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>260</y>
               <width>91</width>
               <height>17</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Total:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="sheer_count_label">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>页</string>
            </attribute>
            <widget class="QTableWidget" name="tableWidget">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>60</y>
               <width>256</width>
               <height>192</height>
              </rect>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutMe"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="separator"/>
   <addaction name="actionMonitor"/>
   <addaction name="actionDebug"/>
   <addaction name="actionTerminal"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionAboutMe"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/icons8-bug-60.png</normaloff>:/images/icons8-bug-60.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/terminal.png</normaloff>:/images/terminal.png</iconset>
   </property>
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutMe">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/aboutme.png</normaloff>:/images/aboutme.png</iconset>
   </property>
   <property name="text">
    <string>AboutMe</string>
   </property>
  </action>
  <action name="actionMonitor">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/computer.png</normaloff>:/images/computer.png</iconset>
   </property>
   <property name="text">
    <string>Monitor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
